adagrams/game.py

```diff
@@ -1,13 +1,152 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+
+}
+
 def draw_letters():
-    pass
+    #empty list to store all the letter
+    all_letter_strings = []
+    #empty list to store ten strings
+    letter_strings_ten = []
+
+    # loop over the dictionary LETTER_POOL
+    for letter, qty in LETTER_POOL.items():
+        while qty > 0: #when qty left > 0
+            all_letter_strings.append(letter) # then append the letter to all_letter_strings, until qty left == 0
+            qty -= 1 #qty left - 1
+
+    while len(letter_strings_ten) < 10: #when letter_strings_ten < 10:
```

Comment:
In Python, we call 10 in this context a "magic number". This means that the number has been hardcoded without much context. If I were a developer unfamiliar with Adagrams, I might not know exactly what that 10 means. We can avoid these magic numbers by making constants with more descriptive names to hold them. In this instance, we know that 10 is the max size of the hand, so we might do something along the lines of:

```suggestion
    MAX_HAND_SIZE = 10
    while len(letter_strings_ten) < MAX_HAND_SIZE: #when letter_strings_ten < 10:
```